x/emissions/keeper/inference_synthesis/network_regrets.go

```diff
@@ -377,15 +380,25 @@ func GetCalcSetNetworkRegrets(args GetCalcSetNetworkRegretsArgs) error {
 		}
 	}
 
+	// Get the initial topic std dev regrets
+	// If we don't have enough workers with enough experience, use the fallback regrets
+	initialTopicStdDevRegrets := make([]alloraMath.Dec, 0)
+	if len(workersRegrets) >= 10 {
+		initialTopicStdDevRegrets = workersRegrets
+	} else if len(fallbackRegrets) > 0 {
+		initialTopicStdDevRegrets = fallbackRegrets
+	}
```

Comment:
This could be simplified
```suggestion
	var initialTopicStdDevRegrets []alloraMath.Dec
	if len(workersRegrets) >= 10 {
		initialTopicStdDevRegrets = workersRegrets
	} else {
		initialTopicStdDevRegrets = fallbackRegrets
	}
```